sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -429,6 +429,34 @@ async Task Verify(Response<BlobDownloadInfo> response)
                 TestHelper.AssertSequenceEqual(data, actual.ToArray());
             }
         }
+
+        [Test]
+        public async Task DownloadAsync_ObjectReplication()
+        {
+            // This is a recorded ONLY test with a special container we previously setup, as we can't auto setup policies yet
```

Comment:
Are ORS policies persistent?  Like if I create an ORS policy, it's not going to go away unless I delete it, correct?

Reply:
Also, is it only possible to apply an ORS policy to a container via the management plane?

Reply:
ORS are policies are persistent. They can only be set and deleted through the management plane

Reply:
Has ORS already been added to the management plane SDK?